src/sbvr-api/compile-auth.ts

```diff
@@ -0,0 +1,51 @@
+import type { Filter, PinejsClientCore, Resource } from 'pinejs-client-core';
+
+const asPermission = (compiledFilter: string): string => {
+	return compiledFilter
+		.split('$filter=')[1]
+		.replace(/Auth\.canAccess\(\)/g, 'canAccess()');
+};
+
+export const getAuthCompiler = <M extends { [key in keyof M]: Resource }>({
```

Comment:
I would be fine adding this to the internal pinejs-client as well so it can be used as `sbvrUtils.api.foo.compileAuth(...)`

Reply:
Yeah, I considered this possibility but my point "agaisnt" it was basically that in most places we use ClientCore/PineClient interchangeably e.g. we sometimes override stuff etc but completely adding new features is something that could be confusing, that's why I went with this approach. And if I think of how I woudl use it in oB/cloudAPI I would end up writing a wrapper e.g. `const resinAuth = sbvrUtils.api.foo.compileAuth(...);` so I went with directly exposing the wrapper